classes/class-wc-connect-shipping-label.php

```diff
@@ -379,14 +379,26 @@ protected function filter_items_needing_shipping( $item ) {
 			return $product && $product->needs_shipping();
 		}
 
+		/**
+		 * Reduce items to sum their quantities.
+		 *
+		 * @param  int   $sum  Current sum.
+		 * @param  array $item Item to add to sum.
+		 * @return int
+		 */
+		protected function reducer_items_quantity( $sum, $item ) {
+			return $sum + $item->get_quantity();
+		}
+
 		public function meta_box( $post, $args ) {
 			$order = wc_get_order( $post );
 			$order_id = WC_Connect_Compatibility::instance()->get_order_id( $order );
 			$items = array_filter( $order->get_items(), array( $this, 'filter_items_needing_shipping' ) );
+			$items = array_reduce( $items, array( $this, 'reducer_items_quantity' ), 0 );
```

Comment:
This would be much more clear if `$items` was renamed here since we are changing its type.
```suggestion
			$items_count = array_reduce( $items, array( $this, 'reducer_items_quantity' ), 0 );
```